sources/myapp.js

```diff
@@ -8,7 +8,7 @@ export default class MyApp extends JetApp {
 			version: VERSION,
 			router: BUILD_AS_MODULE ? EmptyRouter : HashRouter,
 			debug: !PRODUCTION,
-			start: "/top/contacts"
+			start: "/top/settings"
```

Comment:
Suitable to demonstrate the stage3, but not for the app in general. Leave Contacts as the start page.

Reply:
Changed start page to "contacts"